.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main,ci-fix]
```

```suggestion
    branches: [main]
```
Before merging the PR, remove the current branch added for testing CI